mne/tests/test_epochs.py

```diff
@@ -1690,12 +1690,13 @@ def test_split_naming(
     if dst_fpath.parent != tmp_path:
         dst_fpath.parent.mkdir(parents=True)
 
-    epochs.save(dst_fpath, verbose=True, **save_kwargs)
+    split_fnames = epochs.save(dst_fpath, verbose=True, **save_kwargs)
 
     # check that the filenames match the intended pattern
     assert len(list(dst_fpath.parent.iterdir())) == n_files
     assert not (tmp_path / split_fname_fn(n_files)).is_file()
     want_paths = [tmp_path / split_fname_fn(i) for i in range(n_files)]
+    assert np.all(split_fnames == want_paths)
```

Comment:
`split_fnames` should be a list so this should work without any `all`
```suggestion
    assert split_fnames == want_paths
```